build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -202,55 +190,73 @@ rubygem-rake:
+simp-vendored-r10k-gem-fast_gettext:
+  :rpm_name: simp-vendored-r10k-gem-fast_gettext-1.1.2-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-fast_gettext-1.1.2-0.noarch.rpm
+simp-vendored-r10k-gem-gettext:
+  :rpm_name: simp-vendored-r10k-gem-gettext-3.2.9-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-gettext-3.2.9-0.noarch.rpm
+simp-vendored-r10k-gem-gettext-setup:
+  :rpm_name: simp-vendored-r10k-gem-gettext-setup-0.30-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-gettext-setup-0.30-0.noarch.rpm
+simp-vendored-r10k-gem-locale:
+  :rpm_name: simp-vendored-r10k-gem-locale-2.1.2-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-locale-2.1.2-0.noarch.rpm
+simp-vendored-r10k-gem-log4r:
+  :rpm_name: simp-vendored-r10k-gem-log4r-1.1.10-1.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-log4r-1.1.10-1.noarch.rpm
+simp-vendored-r10k-gem-minitar:
+  :rpm_name: simp-vendored-r10k-gem-minitar-0.6.1-1.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-minitar-0.6.1-1.noarch.rpm
+simp-vendored-r10k-gem-multi_json:
+  :rpm_name: simp-vendored-r10k-gem-multi_json-1.12.1-1.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-multi_json-1.12.1-1.noarch.rpm
+simp-vendored-r10k-gem-multipart-post:
+  :rpm_name: simp-vendored-r10k-gem-multipart-post-2.0.0-1.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-multipart-post-2.0.0-1.noarch.rpm
+simp-vendored-r10k-gem-puppet_forge:
+  :rpm_name: simp-vendored-r10k-gem-puppet_forge-2.2.8-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-puppet_forge-2.2.8-0.noarch.rpm
+simp-vendored-r10k-gem-r10k:
+  :rpm_name: simp-vendored-r10k-gem-r10k-2.6.2-1.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-r10k-2.6.2-1.noarch.rpm
+simp-vendored-r10k-gem-semantic_puppet:
+  :rpm_name: simp-vendored-r10k-gem-semantic_puppet-1.0.2-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-semantic_puppet-1.0.2-0.noarch.rpm
+simp-vendored-r10k-gem-text:
+  :rpm_name: simp-vendored-r10k-gem-text-1.3.1-0.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-gem-text-1.3.1-0.noarch.rpm
 sudosh2:
   :rpm_name: sudosh2-1.0.2-2el7.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/sudosh2-1.0.2-2el7.x86_64.rpm
```

Comment:
syslinux-tftpboot: another for CentOS vault

Reply:
Fixed. Needed for el6?